CSS/components/modal.css

```diff
@@ -0,0 +1,39 @@
+.dialog-main{
+    display: none;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-color: rgba(0, 0, 0, 0.5);
+    width: 100vw;
+    height: 100vh;
+}
+.open{
+    display: flex;
+}
+.dailog-btn-wrap{
+    display: flex;
```

Comment:
Same as above, using until you can extract this out

Reply:
Done made the general class of flex in util

Reply:
ya I have general flex class in utils but here I didn't want the user to explicitly type flex so declaring it
is it ok?

Reply:
It's fine but since you already have util class you can use it. Pointed out since in almost all classes in this css , flex is being used. This repetition can be avoided using your util.